index.js

```diff
@@ -53,8 +54,18 @@ const DEFAULTS = {
 };
 
 module.exports = function(options) {
+  if (isFunction(options)) {
```

Comment:
Done in https://github.com/koajs/json-error/commit/61fac8b578c9972b8024aa0ab3d5dc33563f4485